main/src/main/scala/sbt/Keys.scala

```diff
@@ -108,7 +108,8 @@ object Keys {
   @deprecated("Use the cacheDirectory provided by streams.", "0.13.0")
   val cacheDirectory = SettingKey[File]("cache-directory", "Directory used for caching task data.", BMinusSetting)
   val cleanFiles = SettingKey[Seq[File]]("clean-files", "The files to recursively delete during a clean.", BSetting)
```

Comment:
Should this be deprecated? The Scala build customized this setting but since upgrading to 0.13.17 it seems to be ignored and `clean` doesn't clean our custom target directories of `build/quick/classes/<subproject>`

Reply:
Yes. The setting is no longer used.